project-overview.md

```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
 ### Tokens
 
 Many real world attestation solutions and potential extrapolations of their use have been considered to ensure that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+* EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
+* ARM Confidential Computing Architecture (CCA) Token and its composition [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
+* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not adhere to any specific DICE-based protocol). If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.
```

Comment:
```suggestion
* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/).  Note that this does not adhere to any specific DICE-based protocol.  If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.
```

Reply:
Again a editorial view of looking at the same thing, not much different from the text which is currently written. I would retain an explicit note as this does better on emphasis.

Reply:
I think the proposal reads a bit better.  Removing the parenthetical connects the two related sentences ("Note that [...]." and "If you are planning [...]) and makes the flow smoother.  Besides, "note" should be capitalised.